test/integration/api/services_controller_test.rb

```diff
@@ -174,7 +174,7 @@ def update_params
     end
   end
 
-  class BackendApiCreation < Api::ServicesControllerTest
+  class BackendApiCreationTest < Api::ServicesControllerTest
```

Comment:
Unrelated to this PR, but all Test classes should be named ending by `Test` to avoid duplication with the logic classes